### ISSUE TITLE
FAST 1.22 returns 404 after BIG-IP reboot

### ISSUE THREAD

**Issue by simonkowallik** [F5Networks/f5-appsvcs-templates#117]
### Environment
 * FAST RPM Version: 1.22.0
 * BIG-IP Version: 16.1.3.3 (also tested on 16.1.3.2)

### Summary

FAST does not start after rebooting BIG-IP, `/mgmt/shared/fast/info` returns HTTP code `404` as well as:

```json
{
  "code": 404,
  "message": "",
  "referer": "10.1.1.6",
  "errorStack": []
}
```

### Steps To Reproduce

1. Install AS3 3.41.0
2. Wait till `/mgmt/shared/appsvcs/info` returns a `200` (AS3 installed)
3. Install FAST 1.22.0
4. Wait till `/mgmt/shared/fast/info` returns a `200` (FAST installed)
5. Reboot
6. `/mgmt/shared/fast/info` returns `404`
7. Execute `bigstart restart restnoded`
8. `/mgmt/shared/fast/info` returns `200`

**Note:** Sometimes it works after the first reboot and only starts failing later

### Expected Behavior

FAST API endpoint is available after reboots (once BIG-IP started all services).

### Actual Behavior

FAST API endpoints does not start automatically after reboots most of the time.

**Comment by andreykashcheev** [comment]
I was not able to replicate this reported issue using provided repro steps and my BIGIP v16. 
Can you please grab quickview from your box for further investigation? https://my.f5.com/manage/s/article/K12878

Let me know if you have any questions/concerns

**Comment by simonkowallik** [comment]
@andreykashcheev I've send you a direct message with the qkview data.

**Comment by shyawnkarim** [comment]
This issue is now being tracked internally with ID, MYSTIQUE-878.

**Comment by shyawnkarim** [comment]
Closing.

This issue was resolved with [Release 1.24.0](https://github.com/F5Networks/f5-appsvcs-templates/releases/tag/v1.24.0).
